index.html

```diff
@@ -106,10 +106,45 @@ <h2>A project for first-time contributions</h2>
 
             <!-- COPY everything ABOVE this, from contributor card start to end along with the "START" and "END" comment lines -->
             <!-- ============== ^^^^ TEMPLATE ^^^^ ============== -->
+
             <!-- ============================================================== -->
             <!-- DO NOT modify the TEMPLATE directly, make a copy and paste it below -->
             <!-- make sure there is one line of space above and below your card -->
             <!-- ========== ⬇⬇ Paste YOUR CARD directly BELOW this line ⬇⬇ ========== -->
+               <!-- ________ Akira's card START ________  -->
+            <div class="card">
+                <p class="name">Akira Brand</p>
+                <p class="contact">
+                    <i class="fab fa-github"></i>
+                    <a href="www.github.com/akirabrand" target="_blank">@akirabrand</a>
+                </p>
+                <p class="about">
+                    I'm a web developer and opera singer who loves science, tech, business, and video games! 
+                    Sometimes I go outside and garden, too :) 
+                </p>
+                <div class="resources">
+                    <p>3 Useful Dev Resources</p>
+                    <ul>
+                        <li>
+
```

Comment:
Hello @AkiraBrand, thank you for contributing. Please remove this blank line and resubmit for review. Also make sure you run `npm run prettier-html` to make sure your code adheres to the styling requirements. Let us know if you have any questions. Take care 😃